spec/models/spree/order_spec.rb

```diff
@@ -220,8 +220,12 @@
     subject(:order) { Spree::Order.create }
 
     it "should set completed_at" do
-      expect(order).to receive(:touch).with(:completed_at)
-      order.finalize!
+      expect {
+        order.finalize!
+        order.reload
+      }.to change {
+        order.completed_at
+      }.from(nil)
```

Comment:
Nice way to test this. I probably would have gone down the unnecessary task of freezing time and checking it changed `to` the current time..